### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by Gelma** [comment]
@peternewman @larsoner : can you check this out, please?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16281,11 +16282,16 @@ somwhere->somewhere
 somwho->somehow
 somwhow->somehow
 sonething->something
+sooaside->suicide
+soodonim->pseudonym
+sooit->suet
+soop->soup
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Also "scoop", "snoop"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16281,11 +16282,16 @@ somwhere->somewhere
...
 soource->source
 sophicated->sophisticated
 sophmore->sophomore
 sorce->source
 sorceror->sorcerer
+sord->sword
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Also "sore", "sored"

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -16701,11 +16702,16 @@ somwhere->somewhere
 somwho->somehow
 somwhow->somehow
 sonething->something
+sooaside->suicide
+soodonim->pseudonym
+sooit->suet
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest suit?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -16733,6 +16739,7 @@ sould->could, should, sold,
 souldn't->shouldn't
 soundard->soundcard
 sountrack->soundtrack
+sourbraten->sauerbraten
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: German? I don't know what the consensus on this is, @larsoner ?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -16701,11 +16702,16 @@ somwhere->somewhere
...
+sooaside->suicide
+soodonim->pseudonym
+sooit->suet
+soop->soup, scoop, snoop,
 soource->source
 sophicated->sophisticated
 sophmore->sophomore
 sorce->source
 sorceror->sorcerer
+sord->sword, sore, sored,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Possibly suggest sawed?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (arm-in) [reply codespell_lib/data/dictionary.txt]: Seems to exist in English language as a loanword, also
https://en.wikipedia.org/wiki/Sauerbraten

... and there are lots of them.
https://en.wikipedia.org/wiki/Lists_of_English_words_by_country_or_language_of_origin

I guess it is legitimate to include them in the dictionary.

**Pull request merged**